src/neuigkeiten/2021-11-19-release/index.md

```diff
@@ -0,0 +1,91 @@
+---
+draft: false
+layout: layout-neuigkeiten
+date: 2021-11-19
+title: Version 4.27.0 veröffentlicht
+author: Tobias Schneider
+release-version: 4.27.0
+teaserImage: /static/update.png
+---
+
+_Neue Abwesenheitsarten braucht die Urlaubsverwaltung!_
+
+<!-- more -->
+
+## Neue Abwesenheitsarten
+
+Deine Urlaubsverwaltung kann nun weitere Abwesenheiten und du hast die Macht diese freizuschalten!
+
+_Bisher_ gab es in der Urlaubsverwaltung vier Abwesenheitsarten:
+* Erholungsurlaub
+* Sonderurlaub
+* Unbezahlter Urlaub und
+* Überstundenabbau
+
+Diese sind in vielen Fällen ausreichend um deine täglichen Abwesenheiten abzubilden, aber für uns ist ausreichend nicht gut genug.
+Darum haben wir der Urlaubsverwaltung in dieser Version weitere Abwesenheitsarten spendiert und den Begriff "Urlaub" in einigen
+Fällen auch in "Abwesenheiten" umbenannt, sodass auch der Weg der Urlaubsverwaltung zur Abwesenheitsverwaltung deutlich wird.
+Natürlich bleiben wir unserem Namen zunächst treu und die Urlaubsverwaltung wird weiterhin die Urlaubsverwaltung sein. Aber man weiß ja nie was uns die Zukunft bringt.
+
+Die neuen Abwesenheitsarten lassen sich in den Einstellungen mit der Berechtigung 'Office' aktivieren bzw deaktivieren und zusätzlich kann
+konfiguriert werden, ob diese Abwesenheitsart, wie bisher, mit einem Genehmigungsprozess (Einstufig bzw. Zweistufig) versehen ist
```

Comment:
Frag mal Nadine, ich glaube das sind zu viele Kommas :D

Reply:
Der Satz ist auch zu lange, da muss man mittendrin umdenken

Reply:
Beispiel für eine Alternative:

Die neuen Abwesenheitsarten lassen sich in den Einstellungen mit der Berechtigung 'Office' aktivieren bzw deaktivieren. Zusätzlich kann konfiguriert werden, ob für diese Abwesenheitsart der Antrag durch einen Genehmigungsprozess (Einstufig bzw. Zweistufig) abgebildet wird.
Neu ist, dass jede Abwesenheit auch ohne Genehmigungsprozess konfiguriert werden kann.